torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -1383,15 +1383,24 @@ def five_crop_video(
 
 
 def five_crop(
-    inpt: features.ImageOrVideoTypeJIT, size: List[int]
-) -> Tuple[
-    features.ImageOrVideoTypeJIT,
-    features.ImageOrVideoTypeJIT,
-    features.ImageOrVideoTypeJIT,
-    features.ImageOrVideoTypeJIT,
-    features.ImageOrVideoTypeJIT,
+    inpt: Union[features.ImageTypeJIT, features.VideoTypeJIT], size: List[int]
+) -> Union[
+    Tuple[
+        features.ImageTypeJIT,
+        features.ImageTypeJIT,
+        features.ImageTypeJIT,
+        features.ImageTypeJIT,
+        features.ImageTypeJIT,
+    ],
+    Tuple[
+        features.VideoTypeJIT,
+        features.VideoTypeJIT,
+        features.VideoTypeJIT,
+        features.VideoTypeJIT,
+        features.VideoTypeJIT,
+    ],
```

Comment:
JIT [doesn't like](https://github.com/pytorch/vision/actions/runs/3234154080/jobs/5296910400) this:
```
____________ TestDispatchers.test_scripted_smoke[cpu-five_crop-06] _____________
Traceback (most recent call last):
  File "/home/runner/work/vision/vision/test/test_prototype_transforms_functional.py", line 24, in script
    return torch.jit.script(fn)
  File "/opt/hostedtoolcache/Python/3.7.14/x64/lib/python3.7/site-packages/torch/jit/_script.py", line 1344, in script
    qualified_name, ast, _rcb, get_default_args(obj)
RuntimeError: false INTERNAL ASSERT FAILED at "../aten/src/ATen/core/union_type.cpp":212, please report a bug to PyTorch. After type unification was performed, the Union with the original types {Tuple[Tensor, Tensor, Tensor, Tensor, Tensor] Tuple[Tensor, Tensor, Tensor, Tensor, Tensor], } has the single type Tuple[Tensor, Tensor, Tensor, Tensor, Tensor]. Use the common supertype instead of creating a Uniontype

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/runner/work/vision/vision/test/test_prototype_transforms_functional.py", line 221, in test_scripted_smoke
    dispatcher = script(info.dispatcher)
  File "/home/runner/work/vision/vision/test/common_utils.py", line 225, in wrapper
    raise exc_tb[0].with_traceback(exc_tb[1])
  File "/home/runner/work/vision/vision/test/common_utils.py", line 228, in wrapper
    out = fn(*args, **kwargs)
  File "/home/runner/work/vision/vision/test/test_prototype_transforms_functional.py", line 26, in script
    raise AssertionError(f"Trying to `torch.jit.script` '{fn.__name__}' raised the error above.") from error
AssertionError: Trying to `torch.jit.script` 'five_crop' raised the error above.
```